docs/ide-integrations/visual-studio/index.md

```diff
@@ -1,10 +1,28 @@
-# Reqnroll Visual Studio integration
+---
+hide-toc: true
```

Comment:
why hide-toc?

Reply:
I his this one because the entire page is a TOC. To have the auto-generated one is superfluous.